Source/Core/CameraLimiter.js

```diff
@@ -0,0 +1,73 @@
+define([
+    '../Core/Cartesian3',
+    '../Core/Check',
+    '../Core/defaultValue',
+    '../Core/defined',
+    '../Core/HeadingPitchRoll',
+    '../Core/Math'
+], function(
+    Cartesian3,
+    Check,
+    defaultValue,
+    defined,
+    HeadingPitchRoll,
+    CesiumMath) {
+    'use strict';
+
+    /**
+     * Object used to define bounds limiting the {@link @Camera#position} when in 3D mode.
+     *
+     * {@link @Camera#cameraLimiter}
+     *
+     * Used by a {@link Camera} to limit the camera's position.
+     *
+     * @alias CameraLimiter
+     * @constructor
+     *
+     * @param {AxisAlignedBoundingBox|BoundingSphere|OrientedBoundingBox} [options.boundingObject] The BoundingObject used to restrict camera positions of the {@link Camera} holding this CameraLimiter.
+     */
+    function CameraLimiter(options) {
```

Comment:
This sholdn't be an `options` parameter if there's only one.  We use an `options` object when a function takes a ton of different parameters to make it more concise.  See https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/CodingGuide#options-parameters